ts/src/cex.ts

```diff
@@ -839,7 +849,48 @@ export default class cex extends Exchange {
         return await this.privatePostCancelOrder (this.extend (request, params));
     }
 
+    async cancelAllOrders (symbol: string = undefined, params = {}) {
+        /**
+         * @method
+         * @name cex#cancelAllOrders
+         * @see https://docs.cex.io/#cancel-all-orders-for-given-pair
+         * @description cancel all open orders in a market
+         * @param {string} symbol unified market symbol of the market to cancel orders in
+         * @param {object} [params] extra parameters specific to the cex api endpoint
+         * @param {string} [params.marginMode] 'cross' or 'isolated', for spot margin trading
+         * @returns {object[]} a list of [order structures]{@link https://docs.ccxt.com/#/?id=order-structure}
+         */
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' cancelAllOrders requires a symbol.');
+        }
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'pair': market['id'],
+        };
+        const orders = await this.privatePostCancelOrdersPair (this.extend (request, params));
+        //
+        //  {
+        //      "e":"cancel_orders",
+        //      "ok":"ok",
+        //      "data":[
+        //      ]
+        //   }
+        //
+        return orders;
+    }
+
     parseOrder (order, market = undefined) {
+        // ws response
```

Comment:
@pcriadoperez hum the WS update does not have the market in it, if possible we should find a way to detect it somehow

Reply:
Good point. I made a change to parse the order in ther request function so we can use the market from the request.